scripts/obtain/dotnet-install.sh

```diff
@@ -159,6 +164,10 @@ get_linux_platform_name() {
     return 1
 }
 
+is_musl_based_distro() {
+    (ldd 2>&1 || true) | grep -q musl
```

Comment:
Can you explain how running "ldd" determines weather a machine is musl based - ldd requires a parameter on a Ubuntu machine. Have you tested this on Alpine? The parsing for "alpine" "void" "gentoo" in the platform name seems safer.

Reply:
`ldd` without arguments on musl-libc distributions outputs:

```
musl libc (x86_64)
Version 1.1.22
Dynamic Program Loader
Usage: /lib/ld-musl-x86_64.so.1 [options] [--] pathname
```

Generally, to my knowledge, the most portable way to determine libc flavor (glibc, llvm-libc, ulibc, musl-libc etc.) on Unix-like OS is: `ldd --version 2>&1`. Alpine Linux' official package repo also has such example: https://github.com/alpinelinux/aports/blob/bfc00db0e316c32ea41ee272621ca1a24a1fde5e/community/openjdk10/build.patch#L22.